frontend/src/framework/store/middlewares/error-handler/error-handler.middleware.ts

```diff
@@ -0,0 +1,19 @@
+import { type AnyAction, type Middleware } from '@reduxjs/toolkit';
+import { isRejected } from '@reduxjs/toolkit';
+
+import { notificationService } from '~/services/services.js';
+
+const errorHandler: Middleware = function () {
+    return function (next) {
+        return function (action: AnyAction) {
+            if (isRejected(action)) {
+                const message = action.error.message;
+                notificationService.error(message);
```

Comment:
you dont need to call the service directly. Create action in the app/store and call action

Reply:
create another action and call it inside middleware in the way of `dispatch(notificationActions.error())`?

Reply:
error prevent me call dispatch from middleware